config/webpack.common.config.js

```diff
@@ -19,4 +30,16 @@ module.exports = {
     },
     extensions: ['.js', '.jsx'],
   },
+  optimization: {
```

Comment:
I see the various `chunks: ['app']` below, and this optimization block here too.  What's the net effect of how we've changed the chunking?  What do the cacheGroups do for us?

Reply:
Correct, the chunking is staying the same as is it today, with the addition of splitting out the "locally" installed already-compiled CSS files from `@edx/paragon` and `@edx/brand`. I will add a comment :)

That said, we probably should find some ways to optimize our MFE Webpack bundles (e.g., test hypothesis that many small JS files for the `node_modules` is better than 1 larger JS file, recommend/implement a strategy for code splitting for MFEs): https://github.com/openedx/wg-frontend/issues/173